### PR TITLE
Math magicians: fetch data from API

### PR DESCRIPTION
**Pull request by jlcarrascof**
In this milestone, I did the following:

- [ ] Use the following external API: **[API Ninjas](https://api-ninjas.com/api/quotes)**.
- [ ] Create a **new component for displaying a quote**.
- [ ] **Fetch data inside of a new component**.
- [ ] **Show a loading state**.
- [ ] **Show an error state**.
- [ ] **Import the new component** into the main component.
- [ ] **Both components are on the same page**.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 function Calculator() {
   return (
-    <div className="calc">
-      <Buttons />
-    </div>
+    <body>
+      <h1 className="title">Math Magicians</h1>
+      <div className="calc">
+        <Buttons />
+        <QuoteContainer />
```
Comment (gemmen29) [inline src/components/Calculator.js]: - [ ] Kindly import the new component `QuoteContainer` into the main component `App` (next to the Calculator component) instead as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=Show%20an%20error-,state,the%20new%20component%20into%20your%20main%20component%20(next%20to%20the%20Calculator),-It%20should%20be).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @jlcarrascof, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (jlcarrascof) [reply src/components/Calculator.js]: Ohhh boyy 😨😨😨

Yes, you're reason, Mr @gemmen29 I'll work on it 🫡

**Review by fobadara — APPROVED** [review]
Hi @jlcarrascof, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Proper implementation of reviewer's requirements
✅ Clean code

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**